core/sodasql/scan/scan_builder.py

```diff
@@ -68,8 +72,17 @@ def __init__(self):
         self.parsers: List[Parser] = []
         self.assert_no_warnings_or_errors = True
         self.soda_server_client: SodaServerClient = None
+        latest_version = ""
+        try:
+            latest_version = luddite.get_version_pypi('soda-sql-core')
+        except:
+            logger.warning("Cannot check Soda SQL version.")
+
+        if SODA_SQL_VERSION != latest_version:
```

Comment:
This shouldn't trigger if we are not able to get the latest version (in case of running behind a firewall or network issue) otherwise even though people are using latest version the logger will still report that it is not latest.